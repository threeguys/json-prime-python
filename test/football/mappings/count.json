[
    {
        "count": "$[*].competition_gender"
    },
    {
        "count": {
            "filter": "$[*].competition_gender",
            "equals": "female"
        }
    },
    {
        "count": {
            "filter": "$[*].competition_gender",
            "prefix": "m"
        }
    }
]
