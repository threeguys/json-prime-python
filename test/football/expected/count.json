[ 20, 4, 16 ]
